### PR TITLE
added erlang support

### PR DESCRIPTION
**Pull request by markbaas**
Preliminary erlang support. Not sure how I can get erlang to show in the settings though.

Tests will follow

### CODE REVIEW

**Comment by Glavin001** [comment]
Let me know if you have any questions! Good work so far.

**Comment by markbaas** [comment]
Hey @Glavin001 I can't figure how to get Erlang to show in Settings page. Did I miss something somewhere?

File: src/languages/erlang.coffee [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  grammars: [
+    "Erlang"
+  ]
+
+  ###
+  Supported extensions
+  ###
+  extensions: ['erl']
+
+  options: []
```
Comment (Glavin001) [inline src/languages/erlang.coffee]: These options would appear in the Atom Settings View.

**Comment by Glavin001** [comment]
@markbaas do you have some particular options that you want to see in the Atom Settings View?

**Comment by Glavin001** [comment]
Published to v0.28.23

**Pull request merged**